src/pybamm/models/submodels/interface/sei/sei_growth.py

```diff
@@ -170,21 +159,15 @@ def get_coupled_variables(self, variables):
                 name = f"{Domain} EC surface concentration [mol.m-3]"
             variables.update({name: c_ec, f"X-averaged {name}": c_ec_av})
 
-        if SEI_option.startswith("ec reaction limited"):
-            inner_sei_proportion = 0
-        else:
-            inner_sei_proportion = phase_param.inner_sei_proportion
-
         # All SEI growth mechanisms assumed to have Arrhenius dependence
         Arrhenius = pybamm.exp(
```

Comment:
Fixing a merge mistake
```suggestion
        arrhenius = pybamm.exp(
```